### ISSUE TITLE
No Context Menu on top folders

### ISSUE THREAD

**Issue by Majdoddin** [aaFn/Bookmark-search-plus-2#27]
Hello,
When the addon is _not_ active, in the sidebar, I can right click on "Bookmarks Menu", "Mobile Bookmarks", ... and get a context menu. From there I can fore example create a subfolder. 
But when the addon is active, I get no context menu. 
Thanks!

**Comment by aaFn** [comment]
Hello @Majdoddin , indeed, I removed them some time ago to protect those "system" folder, but I guess this was a bit excessive.

I can now look at a reduced menu for them. What about:
```
New bookmark..
New folder..
New separator
------------------
Paste into
```
And of course allow Drag and drop into, which is also disabled today ?

**Comment by Majdoddin** [comment]
Hallo @aaFn, if by protection you mean not to be deleted, `Delete` and `Cut` are already disabled in the default context menu. `Copy` is not disabled, but it seems harmless.
Also, as a user, I can't think of any harm from Drag and Drop between these System folders. 
Thanks!

**Comment by aaFn** [comment]
Ok, I'll see if it is easy and not dangerous to allow drag and drop between them,  in addition :-)

Copy .. I don't like it as it can be huge (some people here have 80000 bookmarks) and misleading (what does that really mean to copy a system folder ? .. result is not a system folder). So for now I shouldn't consider it if you don't mind.

**Comment by Majdoddin** [comment]
You are right.

**Comment by aaFn** [comment]
Reopening the issue to track it properly .. wil try for 2.0.13

**Comment by aaFn** [comment]
Ok, I implemented the above context menu on top protected (system) folders in 2.0.13.

I also allowed drag and drop into those top folders, but not between them, as I observed this is not allowed by the native bookmark sidebar nor the Bookmark library.

By extension, I am not allowing droping besides any protected element (recent tags, recent bookmarks ..), even when not top level. I know the native sidebar allows it, but this is a very special case, not going to happen often, and complexifying the code for no significant benefit at this stage.

2.0.13 should be out later this week, or this week end. I will close the issue at that time.
